### PR TITLE
WIP: Support 53

### PR DESCRIPTION
**Pull request by gggeek**
- still missing the 5.2 sql
- not 100% sure why the original update-5.1 sql was missing some stuff

### CODE REVIEW

File: update/database/ezpublish/5.3/dbupdate-5.2.0-to-5.3.0.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE ezsite_data SET value='5.3.0alpha1' WHERE name='ezpublish-version';
+
+CREATE INDEX ezcontentobject_classattr_id ON ezcontentobject_attribute (contentclassattribute_id);
```
Comment (andrerom) [inline update/database/ezpublish/5.3/dbupdate-5.2.0-to-5.3.0.sql]: nitpick: missing new line

Comment (gggeek) [inline update/database/ezpublish/5.3/dbupdate-5.2.0-to-5.3.0.sql]: fixed

**Comment by gggeek** [comment]
update for 5.2 added.

One last thing to polish: the 5.0 -> 5.1 upgrade has a problem with column definition change (oracle does not like it if you try to nullify a null column or viceversa)

**Comment by gggeek** [comment]
Update for 5.0=>5.1 fixed.
Should be good to go.

**Comment by andrerom** [comment]
+1

**Comment by gggeek** [comment]
More thorough testing (comparison of dump of db after upgrade script and dump of db created from scratch) revealed a couple more quirks.
Everything should be *really* ok now

**Comment by andrerom** [comment]
Btw are we sure customer is on 5.3 or should 5.4 be added while on it as well?

**Comment by andrerom** [comment]
Seems we should add 5.4 as well if that is ok with you.

**Pull request merged**